TUTORIAL.md

```diff
@@ -952,6 +957,38 @@ Mission accomplished!
 Of course substitute the wallpaper for one of your own.  If you like the
 one used above, you can find it [here](https://i.imgur.com/9MQHuZx.png).
 
+### Using the `startupHook`
```

Comment:
I wonder if it's worth mentioning [fbautostart](https://github.com/paultag/fbautostart) here.

Reply:
Mh, the README doesn't say much; what exactly does it do and how does one use it?

Reply:
Yes, the documentation is sadly sparse. It enables use of XDG autostart programs (`*.desktop` files in `/etc/xdg/autostart` and `~/.config/autostart`) without running a full desktop manager.

Reply:
I suppose we should also in that case mention `lxsession-edit` as a way to create and manage them without a full desktop manager.

Reply:
Actually it looks like just "manage"; unlike the MATE version, you can't create them.

Reply:
Also I should mention that the main reason you would do this is that there are applications that automatically install XDG autostart entries, so you would either need to translate those into `startupHook` entries or arrange for XDG autostarts to run for them to work properly. In my case (xmonad under MATE) I don't have to worry about this, since MATE does it, but people who don't use xmonad with a desktop manager may care.

Reply:
Looks like there's also `xdg-autostart` from the `obsession` package, which goes with Openbox by default. But, unlike `fbautostart`, already supports use with other environments (XDG allows you to specify that startup programs can be associated with specific environments instead of always started.) This one might be more generally available, whereas the other seems to have been written specifically for Ubuntu.

Reply:
What are these programs that automatically install XDG autostart entries? I've never seen this before. Also, I can't find any `obsession` package in the usual places (Nix, Arch, … repositories); perhaps that is again something Debian/Ubuntu specific?

Reply:
Google's telling me Debian and Slackware. It looks like `fbautostart` is in AUR, so it may be the more common one. The Arch Wiki also suggests a `dex` package (https://github.com/jceb/dex), which also contains a full editor for them.

The programs that contain autostart entries are generally intended for full desktop environments (mainly Gnome) so maybe this isn't so relevant.

Reply:
Mh, yeah it's perhaps a bit outside of the scope of this PR. It'd be nice if *someone* wrote something relating all of these different programs, but I don't feel like I have the expertise to do so